### ISSUE TITLE
Pagination links are incorrect links when params are passed to the endpoint

### ISSUE THREAD

**Issue by buren** [justarrived/just_match_api#556]
If you request: `jobs?sort=filled,-created-at` the response `links` attribute will be:

```json
"links": {
  "self": "http://localhost:3000/api/v1/jobs?sort=filled,-created-at?page%5Bnumber%5D=1&page%5Bsize%5D=10&sort=filled%2C-created-at",
  "next": "http://localhost:3000/api/v1/jobs?sort=filled,-created-at?page%5Bnumber%5D=2&page%5Bsize%5D=10&sort=filled%2C-created-at",
  "last": "http://localhost:3000/api/v1/jobs?sort=filled,-created-at?page%5Bnumber%5D=3&page%5Bsize%5D=10&sort=filled%2C-created-at"
}
```

notice that the link has `?` twice, so it doesn't work properly if either `sort`, `filter`, `page[number]` or `page[size]` params present in the URL.

**Comment by buren** [comment]
Added the label `frontend-blocker`, since this hinders the frontend to use `links.next`, `links.last` etc.

**Comment by buren** [comment]
Closed by #563
